include/sparrow/data_type.hpp

```diff
@@ -141,6 +141,17 @@ namespace sparrow
     template <class T>
     concept is_arrow_base_type = mpl::contains<T>(all_base_types);
 
+     using all_base_types_extended_t = mpl::typelist<
```

Comment:
Maybe (not sure at all) we could add an `append` metafunction in mp_utils and use that to make this typelist the base list + extensions? (or a third one)
That way we can use that extended list in the tests.

Reply:
I added typelist_append_t